src/test/java/io/openliberty/tools/intellij/it/SingleModLibertyLSTestCommon.java

```diff
@@ -86,7 +86,7 @@ public void testServerXMLFeatureHover() {
         String hoverExpectedOutcome = "This feature provides support for the MicroProfile Health specification.";
 
         //mover cursor to hover point
-        UIBotTestUtils.hoverInAppServerCfgFile(remoteRobot, testHoverTarget, "server.xml", UIBotTestUtils.PopupType.DOCUMENTATION);
+        UIBotTestUtils.hoverInAppServerCfgFile(remoteRobot, testHoverTarget, "server.xml, XML file", UIBotTestUtils.PopupType.DOCUMENTATION);
```

Comment:
These accessible names should be constants. It eliminates potential human error having to write this or copy paste multiple times. It also helps when changes like this are required - only one spot in the code needs to be updated. It might be worth setting that up now.